pom.xml

```diff
@@ -57,9 +57,9 @@
     <pentaho-metaverse.version>8.1-SNAPSHOT</pentaho-metaverse.version>
     <commons.lang.version>2.6</commons.lang.version>
     <pentaho-osgi-bundles.version>8.1-SNAPSHOT</pentaho-osgi-bundles.version>
-    <pentaho-osgi-bundles.osgi.version>8.0.0.SNAPSHOT</pentaho-osgi-bundles.osgi.version>
+    <pentaho-osgi-bundles.osgi.version>8.1-SNAPSHOT</pentaho-osgi-bundles.osgi.version>
     <kettle.version>8.1-SNAPSHOT</kettle.version>
-    <kettle.osgi.version>8.0.0.SNAPSHOT</kettle.osgi.version>
+    <kettle.osgi.version>8.1-SNAPSHOT</kettle.osgi.version>
```

Comment:
osgi.versions must be written without the "-". Karaf is strict when processing the versions from the custom.properties file. If the version is not osgi (e.g. contains a "-" separator) karaf gives an error.